virtualizarr/kerchunk.py

```diff
@@ -86,7 +88,7 @@ def read_kerchunk_references_from_file(
 
         refs = NetCDF3ToZarr(filepath, inline_threshold=0, **reader_options).translate()
 
-    elif filetype.name.lower() == "netcdf4":
+    elif filetype.name.lower() == "hdf5" or filetype.name.lower() == "netcdf4":
```

Comment:
keeping netcdf4 as an alias for hdf5 because i've seen code in reported issues using filetype=netcdf4